contentctl/actions/detection_testing/infrastructures/DetectionTestingInfrastructure.py

```diff
@@ -375,7 +375,20 @@ def execute(self):
                 self.sync_obj.currentTestingQueue[self.get_name()] = None
 
     def test_detection(self, detection: Detection):
-        if detection.tests is None:
+        if detection.tags.manual_test:
+            if len(detection.tests) != 1:
```

Comment:
The edge case of there being 2 or more tests that were added, which I think you'd still want to validate against

Reply:
I believe this this has been addressed. Now, each test under a detection tagged manual_test gets status `skip` and a message describing WHY it was skipped (the text contained in the manual_test field) AND if a detection tagged `manual_test` does not have ANY tests, it gets a descriptive, fake one added at runtime showing that it was a manual_test detection.

Reply:
Sweet! I think it's much cleaner to do the skipping elsewhere; like this change a lot